tests/backstop/scenarios.js

```diff
@@ -36,7 +36,7 @@ const path = require( 'path' );
  */
 const legacyStorybookScenarios = require( '../../.storybook/storybook-data' );
 const storybookConfig = require( '../../.storybook/main' );
-const rootURL = '/dist/iframe.html?id=';
+const rootURL = 'file:///src/dist/iframe.html?id=';
```

Comment:
This fixes a notice from being triggered on every story where it says something like "relative URL detected" which makes for unnecessarily verbose output.